app/infrastructure/persistence/repositories/document_repo.py

```
@@ -4,10 +4,11 @@
 
 from typing import TYPE_CHECKING, Any
 
-from sqlalchemy import and_, select, update
+from sqlalchemy import and_, or_, select, update
```

Flake8 (7.3.0)

[error] 'sqlalchemy.or_' imported but unused

(F401)